src/components/authorization/policies/policies.ts

```diff
@@ -165,7 +165,7 @@ const policies = [
     modifiedAt: Perm.ReadAndEdit,
     engagement: Perm.ReadAndEdit,
     primaryLocation: Perm.ReadAndEdit,
-    nonPrimaryLocation: Perm.ReadAndEdit,
+    otherLocations: Perm.ReadAndEdit,
```

Comment:
@michaelmarshall Are you ok with this name change? Does it need to be reflected in DB schema doc?

Reply:
yes, it is updated. Thanks for making sure.